### PR TITLE
Navigation and enhanced reponsive design

### PR DESCRIPTION
**Pull request by Lenkly**
I made some changes in the appearance of the app, so that it can be viewed on a desktop screen without looking odd.
I also put a navigation on the details page to get back to the matchlist, and added some other navigation bars as well for later use. A profile page will be accessable when I add login functionality.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -15,15 +16,18 @@ function App() {
           <Route exact path="/">
             <Welcome />
           </Route>
-          <Route exact path="/genres">
+          <Route exact path="/Genres">
```
Comment (lmachens) [inline client/src/App.js:19]: I would revert these changes. paths are usually in lower case (see other pages like reactjs.org).

File: client/src/components/CalcIcon.js [hunk 2]
```diff
@@ -10,19 +10,56 @@ const CalcIcon = styled.button`
...
-  average: { bordercolor: '#59ff00', textcolor: '#59ff00' },
-  okay: { bordercolor: '#00ff6a', textcolor: '#00ff6a' },
-  substandard: { bordercolor: '#00ccff', textcolor: '#00ccff' },
-  low: { bordercolor: '#006AFF', textcolor: '#006AFF' },
-  verylow: { bordercolor: '#bb00ff', textcolor: '#bb00ff' },
-  almostnone: { bordercolor: '#7700ff', textcolor: '#7700ff' },
+  perfect: {
+    bordercolor: '#ff0084',
+    textcolor: '#ff0084',
+    shadowcolor: '#ff0084',
```
Comment (lmachens) [inline client/src/components/CalcIcon.js:20]: are these colors always the same?
-> reduce it to a single property.

Besides that, this is a case for camelCase :).

Comment (Lenkly) [reply client/src/components/CalcIcon.js]: you are right, reduced this to the minimum :)
